src/styles/colour/index.md.njk

```diff
@@ -45,70 +48,40 @@ the GOV.UK colour palette when you update GOV.UK Frontend.
         <td class="app-colour-list-column app-colour-list-column--notes">
           {{colour.notes}}
         </td>
-
       </tr>
     {% endfor %}
-    
-
   {% endfor %}
   </tbody>
 </table>
 
-## Greyscale
-
-<table class="govuk-body app-colour-list" summary="Table of greyscale colours"> 
-
-  <tbody>
-  {% for colour in colours.greyscale %}
-
-    <tr class="app-colour-list-row">
-
-      <th class="app-colour-list-column app-colour-list-column--name">
-        <span class="app-swatch {% if colour.colour == "#ffffff" %}app-swatch-border{% endif %}" style="background-color:{{colour.colour}}"></span>
-        <code>{{colour.name}}</code>
-      </th>
-
-      <td class="app-colour-list-column app-colour-list-column--colour">
-        {{colour.colour}}
-      </td>
-      <td class="app-colour-list-column app-colour-list-column--notes">
-        {{colour.notes}}
-      </td>
-
-    </tr>
+## Colour palette
 
-  {% endfor %}
-  </tbody>
-</table>
+Use these colours for graphs and supporting material.
 
-## Extended colours
+To reference colours from the palette directly you should use the `govuk-colour`
+function.
 
-Use these colours for graphs and supporting material.
+Avoid using the palette colours directly when there is a contextual colour that
+meets your use case. For example, if you are styling a component's error state
+you should use `$govuk-error-colour` rather than `govuk-colour("red")`.
```

Comment:
```suggestion
use the `$govuk-error-colour` Sass variable rather than `govuk-colour("red")`.
```

Reply:
It might be just me, but I think of these as semantic colours rather than contextual colours.

Reply:
This sentence has changed completely since this comment – it now reads:

> Avoid using the palette colours if there is a Sass variable that is designed for your context. For example, if you are styling the error state of a component you should use the `$govuk-error-colour` Sass variable rather than `govuk-colour("red")`.

I started out with 'semantic' but I'm not sure how well understood that term is generally. Something for us to look out for in user research 👍